frontend/components/puzzle1.jsx

```diff
@@ -20,13 +27,18 @@ class Puzzle1 extends React.Component {
     var userinput = this.state.value;
     this.props.send('submit', { puzzle: '1', answer: userinput });
   }
+
+  cipherPing() {
+    this.props.ping('cipher_ping', { 'hello': 'hello' });
+  }
   render() {
     const { value } = this.state;
     const { correct } = this.props;
     return (
       <div>
         <PuzzleHeader title="Puzzle One" />
         <div id="cipher-container">
+          {this.state.cipher}
```

Comment:
we're already deconstructing at the top of this function, let's keep things consistent